pages/static/javascripts/authform.js

```diff
@@ -0,0 +1,54 @@
+$(document).ready(function(){
+  $("#login-form-js").on("submit", function(){
+    $this = $(this)
+    $.ajax({
+      url: $this.attr("action"),
+      method: $this.attr("method"),
+      data: $this.serialize()
+    }).done(function(data){
+      document.location = data.location
+    }).fail(function(data){
+      errorContainer = $("#login-form-errors-js");
+      errorContainer.html("");
+      errors = []
+      if(data.responseJSON.form_errors.__all__)
+        errors = errors.concat(data.responseJSON.form_errors.__all__);
+      if(data.responseJSON.form_errors.email)
+        errors = errors.concat(data.responseJSON.form_errors.email);
+      if(data.responseJSON.form_errors.password)
+        errors = errors.concat(data.responseJSON.form_errors.password);
+      console.log(errors);
```

Comment:
@dethos Debug print, we may keep this, but I'm not sure this was something you missed.